Updated for latest 0.7.6 release

csmith-palantir opened the pull request:
0.7.6 was released to fix the beta expiry.  Updated for this version.
Removed appcast.  Feed points to an html page now.

fanquake commented:
Can you please update the commit message to mention the cask being updated. i.e "Update Shortcat to 0.7.6"

csmith-palantir commented:
Updated the message.  It now reads.

Updated for latest 0.7.6 release

Updated Shortcat to 0.7.6.  0.7.6 was released to fix the beta expiry.
Removed appcast.  Feed points to an html page now.

Charles

From:  Michael <notifications@github.com>
Reply-To:  caskroom/homebrew-cask
<reply+0060fc3acdfe320e2b4c6d7f6707fd528221590a7443b37692cf00000001118603369
2a169ce05054523@reply.github.com>
Date:  Tuesday, June 2, 2015 at 4:50 PM
To:  caskroom/homebrew-cask <homebrew-cask@noreply.github.com>
Cc:  Charles Smith <csmith@palantir.com>
Subject:  Re: [homebrew-cask] Updated for latest 0.7.6 release (#11536)

Can you please update the commit message to mention the cask being updated.
i.e "Update Shortcat to 0.7.6"


Reply to this email directly or view it on GitHub
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_caskroom_ho
mebrew-2Dcask_pull_11536-23issuecomment-2D108134031&d=BQMCaQ&c=izlc9mHr637UR
4lpLEZLFFS3Vn2UXBrZ4tFb6oOnmz8&r=Phq94JWNO4bawESM66szU3oQMY1OAiSn0CxSbbBACnU
&m=pt74yjt8gJB3q2WRG3BumJCw8xwbG1mSSZDnYHULip0&s=Rb4wvc9OvNLdoiOiVZYwH8_OsVn
UdWLXgA_1PuBDo_A&e=> .

alebcay commented:
The commit message still reads "Updated for latest 0.7.6 release", not "Update Shortcat to 0.7.6". Also, could you please [squash your commits](http://davidwalsh.name/squash-commits-git) after changing the commit message?
